recipe/migrations/isl021.yaml

```diff
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+isl:
+- '0.21'
+migrator_ts: '1584154098'
```

Comment:
Isn't this supposed to be an int?

Reply:
Maybe? I'll fix it on the bot either way.

Reply:
I think it can actually be a float, we just didn't bother because it was annoying to have such large numbers (and unneeded when humans were putting up the PRs)